packages/core/src/__tests__/memdb.test.ts

```diff
@@ -19,35 +19,33 @@ import { ModelDb } from '../memdb'
 import core from '../component'
 import { Domain } from '../classes'
 
-describe('hierarchy', () => {
+const txes = require('./core.tx.json') // eslint-disable-line
```

Comment:
I believe you need to return it `as Tx[]` to avoid eslint errors below. Also please specify rule your are disable with // eslint-disable-line

Reply:
added as Tx[], rule specified.